File: src/ruby/square_pattern.rb
def square_pattern(n)
    for i in 1..n do
        for j in 1..n do
            print('* ')
        end
        puts('')
    end
end

if ARGV.length > 1
    puts('Too many arguments!')
else 
    number = ARGV[0]
    square_pattern(number.to_i)    
end

# ruby squarepattern.rb 5
#   * * * * * 
#   * * * * * 
#   * * * * * 
#   * * * * * 
#   * * * * * 

File: src/python/squarepattern.py

def square_pattern(n):
  for i in range(n):
    for j in range(n):
      print("*",end=" ")
    print()

square_pattern(5)

'''
python3 squarepattern.py          

* * * * * 
* * * * * 
* * * * * 
* * * * * 
* * * * *
'''